test/es-module/test-esm-loader-chaining.mjs

```diff
@@ -253,6 +253,23 @@ describe('ESM: loader chaining', { concurrency: true }, () => {
     assert.strictEqual(code, 0);
   });
 
+  it('should allow loaders to influence subsequent loader resolutions', async () => {
+    const { code, stderr } = await spawnPromisified(
+      execPath,
+      [
+        '--loader',
+        fixtures.fileURL('es-module-loaders', 'loader-resolve-strip-xxx.mjs'),
+        '--loader',
+        `xxx/${relative(process.cwd(), fixtures.path('es-module-loaders', 'loader-resolve-strip-yyy.mjs'))}`,
+        ...commonArgs,
+      ],
+      { encoding: 'utf8' },
```

Comment:
This is probably safer as we don't know if the folder where the repo is cloned contains chars that need URL-encoding.
```suggestion
        'xxx/loader-resolve-strip-yyy.mjs',
        ...commonArgs,
      ],
      { encoding: 'utf8', cwd: fixtures.path('es-module-loaders') },
```

Reply:
Updated 👍